testcases/android/master.jinja2

```diff
@@ -0,0 +1,107 @@
+
+{# variables needs to be converted to be used by master.jinja2 #}
+{% set lava_job_name = "lkft-android-" + KERNEL_BRANCH + '-' + BUILD_NUMBER + '-' + job_name %}
+{% set lava_device_type = lava_device_type %}
+{% set LAVA_JOB_PRIORITY = TEST_LAVA_JOB_PRIORITY %}
+{% set job_timeout = timeout_job_total %}
+
+{########## LAVA JOB DEFINITION ##############}
+{% block visibility %}
+  group:
+    - {{TEST_LAVA_JOB_GROUP}}
+{% endblock visibility %}
+
+{% block metadata %}
+  android.build: "{{BUILD_NUMBER}}"
+  android.name: "{{JOB_NAME}}"
+  android.url: "{{REFERENCE_BUILD_URL}}"
+  android.version: "{{ANDROID_VERSION}}"
+{% if ANDROID_VENDOR_FINGERPRINT is defined %}
+  android.build.vendor.fingerprint: "{{ANDROID_VENDOR_FINGERPRINT}}"
+{% endif %}
+{% if ANDROID_GSI_FINGERPRINT is defined %}
+  android.build.gsi.url: "{{ANDROID_GSI_URL}}"
+  android.build.gsi.fingerprint: "{{ANDROID_GSI_FINGERPRINT}}"
+{% endif %}
+{% if TUXSUITE_DOWNLOAD_URL is defined %}
+  tuxsuite.download.url: "{{TUXSUITE_DOWNLOAD_URL}}"
+{% endif %}
+  lkft.build.config: "{{LKFT_BUILD_CONFIG}}"
+  git branch: "{{KERNEL_BRANCH}}"
+  git repo: "{{KERNEL_REPO}}"
+  git commit: "{{KERNEL_COMMIT}}"
+  git describe: "{{KERNEL_DESCRIBE}}"
+  build-url: "{{BUILD_URL}}"
+  toolchain: "{{TEST_METADATA_TOOLCHAIN}}"
+  series: lkft
+{% block job_specific_metadata %}
+{% endblock job_specific_metadata %}
+{% endblock metadata %}
+
+{% block secrets %}
+secrets:
+  ARTIFACTORIAL_TOKEN: "{{ARTIFACTORIAL_TOKEN}}"
+{% endblock %}
+
+actions:
+{% block actions %}
+
+{% block download_target %}
+{% endblock download_target %}
+
+{% block actions_between_download_deploy %}
+{% endblock actions_between_download_deploy %}
+
+{% block deploy_target %}
+{% endblock deploy_target %}
+
+{% block actions_between_deploy_boot %}
+{% endblock actions_between_deploy_boot %}
+
+{% block boot_target %}
+{% endblock boot_target %}
+
+{% block test_common_actions %}
+{% with timeout_tests=timeout_common_tests %}
+{% include "include/android-test-target.jinja2" %}
+{% endwith %}
+    - from: inline
+      path: android-boot.yaml
+      name: android-boot
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: android-boot
+          description: "android-boot"
+        run:
+          steps:
+            - lava-test-case "android-boot-sleep-3mins" --shell sleep 180
+            - lava-test-case "android-boot-lsusb-v" --shell lsusb -v
+            - lava-test-case "android-boot-wait-for-device" --shell adb wait-for-device
+            - lava-test-case "android-boot-sleep-1mins" --shell adb shell sleep 60
+            - lava-test-case "android-boot" --shell adb shell getprop sys.boot_completed
+            - lava-test-case "android-boot-screencap" --shell adb shell screencap -p /data/local/tmp/screencap.png
+{% endblock test_common_actions %}
+
+{% block test_target %}
+{% include "include/android-test-target.jinja2" %}
```

Comment:
this file and include/test_target.jinja2 isn't that different.

Reply:
but don't you think include/test_target.jinja2 is a little over complex to be read and to be used?

Reply:
no.

Reply:
updated to change  include/test_target.jinja2 to support the docker method now.